apps/etterlatte-saksbehandling-ui/client/src/components/oppgavebenk/oppgaveModal/KommentarLabel.tsx

```diff
@@ -5,7 +5,8 @@ export const KommentarLabel = () => (
   <HStack gap="1">
     Kommentar
     <HelpText>
-      Legg til kommentar hvis du avslutter oppgaven. Det er ikke nødvendig dersom du oppretter revurdering.
+      Legg til kommentar hvis du avslutter oppgaven. Dette er ikke nødvendig dersom du oppretter en revurdering eller
```

Comment:
Det blir ikke helt rett, siden vi bruker `KommentarLabel` to steder, og de oppretter enten forbehandling eller revurdering

Reply:
Denne teksten adresserer jo begge tilfellene "... en revurdering eller forbehandling" så jeg skjønner ikke hva du mener med at dette ikke blir helt rett 🤔

Reply:
Kunen det har vært `Du trenger kun å legge inn kommentar hvis du velger å avslutte oppgaven`

Reply:
Du har rett, github gir bare preview på det du har endret.. så såg ikke forbehandling var del av teksten

Reply:
Opprettet et kort med dine forslag, Emil: https://jira.adeo.no/browse/EY-5403